floris/tools/floris_interface.py

```diff
@@ -859,119 +852,37 @@ def get_farm_AEP(
         # Copy the full wind speed array from the floris object and initialize
         # the the farm_power variable as an empty array.
         wind_speeds = np.array(self.floris.flow_field.wind_speeds, copy=True)
-        farm_power = np.zeros((self.floris.flow_field.n_wind_directions, len(wind_speeds)))
+        wind_directions = np.array(self.floris.flow_field.wind_directions, copy=True)
+        farm_power = np.zeros(self.floris.flow_field.n_findex)
 
         # Determine which wind speeds we must evaluate in floris
         conditions_to_evaluate = wind_speeds >= cut_in_wind_speed
         if cut_out_wind_speed is not None:
             conditions_to_evaluate = conditions_to_evaluate & (wind_speeds < cut_out_wind_speed)
 
         # Evaluate the conditions in floris
+        #TODO confirm this is working in new framework
```

Comment:
Is this still an open question?

Reply:
Yes, this is resolved, wrote a test to confirm all works as expected